.github/libs/GitUtils.js

```diff
@@ -122,7 +122,7 @@ function getPullRequestsMergedBetween(fromTag, toTag) {
         // Find which commit messages correspond to merged PR's
         const pullRequestNumbers = getValidMergedPRs(commitList);
         console.log(`List of pull requests merged between ${fromTag} and ${toTag}`, pullRequestNumbers);
-        return pullRequestNumbers;
+        return _.map(pullRequestNumbers, (num) => Number.parseInt(num, 10));
```

Comment:
NAB we could update `getValidMergedPRs` to return `Array<Number>` instead of `Array<String>`

Reply:
good suggestion. Done!